doc/_templates/autosummary.rst

```diff
@@ -7,7 +7,7 @@
 {% if objtype in ['class'] %}
 .. auto{{ objtype }}:: {{ objname }}
     :show-inheritance:
-    :special-members:
+    :special-members: __call__
```

Comment:
```suggestion
    :special-members: __call__,__init__
```

Reply:
This is not great. We don't want all the `__init__` in the docs (https://57666-1385122-gh.circle-artifacts.com/0/doc/build/html/search.html?q=__init__). It's information is already available in the class docstring, e.g.

![grafik](https://user-images.githubusercontent.com/2836374/116478095-d9008380-a87d-11eb-9801-31795948b68d.png)